src/components/app-top-bar.js

```diff
@@ -191,8 +207,20 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
             fetchAppsAndUrls().then((res) => {
                 setAppsAndUrls(res);
             });
+            if (studyUuid !== null) {
+                fetchNetworkModificationTree(studyUuid)
+                    .then((res) => {
```

Comment:
not sure it's the role of the TopBar Component to fetch NetworkModificationTree...